### PR TITLE
Allow the configure task to target other app names

### PR DESCRIPTION
**Pull request by commuterjoy**
```
# configure based on package.json in cwd
node ./bin/next-build-tools configure

# configure engels
node ./bin/next-build-tools configure ft-next-engels

# configure ft-next-engels-pr-1 with ft-next-engels' configuration
node ./bin/next-build-tools configure ft-next-engels ft-next-engels-pr-1 
```

### CODE REVIEW

**Comment by matthew-andrews** [comment]
:+1: yup make sense

File: bin/next-build-tools.js [hunk 1]
```diff
@@ -34,10 +34,10 @@ program
 		});
 
 	program
-		.command('configure')
+		.command('configure [source] [target]')
```
Comment (matthew-andrews) [inline bin/next-build-tools.js]: ah wait

Comment (matthew-andrews) [inline bin/next-build-tools.js]: should this be `option('--source')`?  not that familiar with commander.js

Comment (matthew-andrews) [inline bin/next-build-tools.js]: no actually i  like your api better

**Comment by matthew-andrews** [comment]
:+1:

File: bin/next-build-tools.js (hunk 1 above)
Comment (commuterjoy) [inline bin/next-build-tools.js]: Yup

square brackets - optional, chevrons - required

**Pull request merged**